relay-base-schema/src/data_category.rs

```diff
@@ -84,10 +84,6 @@ pub enum DataCategory {
     /// and metric cardinality. Defined here so as not to clash with future
     /// categories.
     MetricSecond = 19,
-    /// Replay Video
-    ///
-    /// This is the data category for Session Replays produced via a video recording.
-    ReplayVideo = 20,
```

Comment:
I think we shouldn't remove it, we can rename it but we should keep some indicator around that the id `20` was in use and should not be re-used.

Maybe a variation of `DoNotUseReplayVideo` and `#[deprecated]` on the enum variant?